dmoj/monitor.py

```diff
@@ -57,6 +57,14 @@ def __init__(self, refresher=None):
         self.callback = None
 
     def on_any_event(self, event):
+        ALLOWED_EVENT_TYPES = [
```

Comment:
Small nit: could we move this to inside `class SendProblemsHandler`, and have it be a tuple to indicate its immutability? Otherwise LGTM.

Reply:
Instead of tuple can we have a frozenset?

Reply:
On second thought `tuple` is fine given that it's just 4 elements.